templates/layout/page.html.twig

```diff
@@ -0,0 +1,168 @@
+ * - page.top_second: Items for the second top region.
+ * - page.top_third: Items for the third top region.
+ * - page.breadcrumb: Items for the breadcrumb region.
+ * - page.content: The main content of the current page.
+ * - page.sidebar_first: Items for the first sidebar.
+ * - page.sidebar_second: Items for the second sidebar.
+ * - page.bottom_widget_1: Items for the first bottom widget region on the front page.
+ * - page.bottom_widget_2: Items for the second bottom widget region on the front page.
+ * - page.bottom_widget_3: Items for the third bottom widget region on the front page.
+ * - page.footer_first: Items for the first footer region.
+ * - page.footer_second: Items for the second footer region.
+ * - page.footer_third: Items for the third footer region.
+ * - page.footer_fourth: Items for the fourth footer region.
+ * - page.footer: Items for the bottom footer region.
+ *
+ * @see template_preprocess_page()
+ * @see html.html.twig
+ */
+#}
+<div id="header_wrapper">
+  <div id="inner_header_wrapper">
+    <header id="header">
+      <div class="top_left">
+        {{ page.header }}
+      </div>
+      <div id="headerHero" class="top_right">
+        {{ page.search }}
+        <button class="burger-toggle collapse" type="button" data-toggle="collapse" data-target=".main-menu">
+          <span class="sr-only">Menu</span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </button>
+      </div>
+      <div class="clear"></div>
+    </header>
+  </div>
+  {% if page.primary_menu %}
+    {{ page.primary_menu }}
+  {% endif %}
+</div>
+
+{% if is_front %}
+  {{ page.slideshow }}
+  {{ page.front_welcome }}
+
+  {% if page.top_first or page.top_second or page.top_third %}
+    <div id="top-area" class="clearfix">
+      <div class="column">
+        {{ page.top_first }}
+      </div>
+      <div class="column">
+        {{ page.top_second }}
+      </div>
+      <div class="column">
+        {{ page.top_third }}
+      </div>
+    </div>
+  {% endif %}
+{% endif %}
+
+<div id="container">
+  <div class="container-wrap">
+    <div class="content-sidebar-wrap">
+      <div id="content">
+        {{ page.breadcrumb }}
+        {% if page.content %}
+          <main id="post-content">
+            {{ page.content }}
+          </main>
+        {% endif %}
+      </div>
+      {% if page.sidebar_first %}
+        <aside id="sidebar-first">
+          {{ page.sidebar_first }}
+        </aside>
+      {% endif %}
+      {% if page.sidebar_second %}
+        <aside id="sidebar-second">
+          {{ page.sidebar_second }}
+        </aside>
+      {% endif %}
+
+      {% if is_front %}
+        {% if page.bottom_widget_1 or page.bottom_widget_2 or page.bottom_widget_3 %}
+          <div id="bottom_wrapper" class="footer_block bottom_widget">
+            <div id="bottom-area" class="clearfix">
+              <div class="column">
```

Comment:
conditionally render the `div.column` only if there is content for the subsequent region which it is wrapping